test/integration/service/service_test.go

```diff
@@ -631,6 +632,7 @@ func Test_TrafficDistribution_FeatureGateEnableDisable(t *testing.T) {
 	////////////////////////////////////////////////////////////////////////////
 
 	server1.TearDownFn()
+	featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.32"))
```

Comment:
ah, and we don't need any changes for the endpointslice controller unit tests, because they don't use the feature gate directly; the controller just tells the `k8s.io/endpointslice` code whether or not to use the feature